src/components/tutorials/advanced-tutorial.js

```diff
@@ -16,24 +16,23 @@ class AdvancedTutorial extends React.Component {
       <div>
         <div>
           <div className='temp-grid'>
-            <h1 className='ds-h1'>Advanced API Tutorial</h1>
-            <p>In the <a href='/qpp-submissions-docs/tutorial'>first tutorial</a> we covered how to create a submission, add a measurement set with IA category performance data, and retrieve the score in three different API requests. This time we're going to build on the previous tutorial and look at creating a submission with embedded ACI performance data in one request, go through ACI scoring and how measures compose a score, and see how to update a measure with new info (all while running into a problem along the way). All of these examples serve to illustrate how the Submissions API can make it easier to react to and fix issues that arise.</p>
+            <h1 className='ds-h1'>Tutorial: Update Data</h1>
+            <p>In the <a href='/qpp-submissions-docs/tutorial'>first tutorial</a> we covered how to create a measurement set with category performance data, and retrieve the score in three different API requests. This time we're going to build on the previous tutorial look at adding more Quality performance data to the measurement set we created in the previous tutorial (all while running into a problem along the way). All of these examples serve to illustrate how the Submissions API can make it easier to react to and fix issues that arise.</p>
             <p>
           Like the previous tutorial, please remember that the score calculation may be inaccurate until the API is finalized. The <a href='https://qpp-submissions-sandbox.navapbc.com/'>Interactive Docs</a> will always return the latest score calculation.</p>
```

Comment:
Update this paragraph to:

<p>Like the previous tutorial, please remember that the score calculation in this tutorial may not be up to date. The <a href='https://qpp-submissions-sandbox.navapbc.com/'>Interactive Docs</a> will always return the latest score calculation.</p>